internal/cmd/init/init.go

```diff
@@ -205,3 +213,12 @@ func getNodeProjectLang() (config.FuzzTestType, error) {
 	}
 	return config.FuzzTestType(userSelectedLang), nil
 }
+
+// map of supported test types/build systems for init command
+var supportedInitTestTypes = map[string]string{
+	"cmake":  string(config.BuildSystemCMake),
+	"maven":  string(config.BuildSystemMaven),
+	"gradle": string(config.BuildSystemGradle),
+	"js":     string(config.BuildSystemNodeJS),
+	"ts":     string(config.BuildSystemNodeJS),
```

Comment:
My IDE complains that this is a redundant type conversion

```suggestion
	"cmake":  config.BuildSystemCMake,
	"maven":  config.BuildSystemMaven,
	"gradle": config.BuildSystemGradle,
	"js":     config.BuildSystemNodeJS,
	"ts":     config.BuildSystemNodeJS,
```